### PR TITLE
Makes god clashes probably more fair

### LINKED ISSUE #20194: Ratvar (almost[?]) always wins against Narsie
In the code that lets ratvar and narsie fight ratvar makes his attempt to "win" first every time. If he wins, and he always seems to,  the fight ends before poor nar-nar can do anything.

Just saw an adminspawned ratvar with no followers win against a naturally spawned narsie with plenty of them.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Mmmmmmaybe fixes #20194?

This should hopefully made god clashes less likely to just instantly cause one god to win unless that god has a TON of followers, though upsets are possible.

### CODE REVIEW

**Comment by evsey9** [comment]
Ratvar must win

File: code/game/gamemodes/clock_cult/clock_ratvar.dm [hunk 1]
```diff
@@ -289,40 +289,36 @@
...
 			shake_camera(M, 4, 3)
-		if(prob(base_victory_chance + n_success_modifier))
-			winner = "Nar-Sie"
+		if(winner == "Nar-Sie")
 			break
-		base_victory_chance++ //The clash has a higher chance of resolving each time both gods attack one another
+		base_victory_chance *= 2 //The clash has a higher chance of resolving each time both gods attack one another
 	switch(winner)
 		if("Ratvar")
-			world << "<span class='heavy_brass'><font size=5>\"[pick("DIE! DIE! DIE!", "REEEEEEEEE!", "FILTH!!!", "SUFFER!!!", text2ratvar("ROT FOR CENTURIES AS I HAVE!!"))]\"</font></span>" //nar-sie get out
```
Comment (Iamgoofball) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: no

Comment (PKPenguin321) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: reeeeeeeeeeeeeeeeee

Comment (optimumtact) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: bundling unrelated changes is so passe

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: get your reeeee out of here

Comment (kilkun) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: would make it 100% better if they didn't speak english at all
